src/myframe.h

```diff
@@ -3,14 +3,14 @@ struct MyFrame : wxFrame {
     wxString exepath_;
     wxFileHistory filehistory;
     wxTextCtrl *filter, *replaces;
-    wxToolBar *tb;
+    wxAuiToolBar *tb1, *tb2, *tb3, *tb4, *tb5, *tb6, *tb7, *tb8, *tb9; 

```

Comment:
give these an actual name of whats in them?

Reply:
:-) Oh, you are completely right! I corrected this and pushed an additional commit for it in this PR.